manifests/plugin/python.pp

```diff
@@ -1,45 +1,51 @@
 # See http://collectd.org/documentation/manpages/collectd.conf.5.shtml#plugin_python
-define collectd::plugin::python (
-  $modulepath,
-  $module,
-  $script_source,
-  $ensure = present,
-  $config = {},
-  $order = '10',
+class collectd::plugin::python (
+  $modulepath = '/usr/lib/collectd/python',
+  $ensure     = present,
+  $modules    = {},
+  $order      = '10',
```

Comment:
Is this parameter useful in a class context ? (the only other case where we have this is for the perl module)